skillbridge/src/component/BenefitSection/BenefitSection.tsx

```diff
@@ -1,10 +1,57 @@
 import React from 'react'
 import { Benefitlist } from '../Data/BenefitData'
 import { benefitIcon } from '../../assets/resource/iconResource';
-// import '../BenefitSection/BenefitSection.css'
-
+// import '../BenefitSection/BenefitSection.scss'
+import useToggleState from '../CommonFunctionalities/Custom Hook/useToggleState';
 
 function BenefitSection() {
+    
+    const [viewAll, setViewAll] = useToggleState(false);
+
+    const RenderBenefitSection=() => {
+        return(
+            <div className='d-flex flex-column gap-2'>
+            <div className="row gap-1">
+                {Benefitlist.filter(benefit => {
+                    return (benefit.id <= 3);
+                }).map((benefit) => (
+                        <div  className="col-md  benefit_sub_container bg-white   p-4 rounded ms-1"
+                             key={benefit.id}>
+                            <div className="number_container d-flex justify-content-end"><h1>{benefit.benefit_number}</h1></div>
+                            <div className="benefit_text_container">
+                                <h4>{benefit.title}</h4>
+                                <p>{benefit.desc}</p>
+                            </div>
+                            <div
+                                className="benefit_button_container d-flex justify-content-end">
+                                <button className="rounded border-0 "><img
+                                    src={benefitIcon} alt='benefitImage'/></button>
+                            </div>
+                        </div>
+                ))}
+            </div>
+            <div className="row gap-1">
+                {Benefitlist.filter(benefit => {
+                    return (benefit.id >3);
+                }).map((benefit) => (
```

Comment:
same comment as before and also you are repeating this filter twice in the same file which should be avoided